src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/FlagParser.cs

```diff
@@ -27,13 +73,20 @@ public static Flags Parse(string flagString)
             foreach (string token in tokens)
             {
                 var hasPayload = token.Split('=');
+
+                FlagType flag;
+                if (!_ruleFlagLookup.TryGetValue(hasPayload[0], out flag))
+                {
+                    throw new FormatException("Unrecognized flag: " + hasPayload[0]);
```

Comment:
Nit: We begin to use [string interpolation(https://msdn.microsoft.com/en-us/library/dn961160.aspx)] commonly now.